components/director/internal/domain/labeldef/service.go

```diff
@@ -60,3 +75,52 @@ func (s *service) List(ctx context.Context, tenant string) ([]model.LabelDefinit
 	}
 	return defs, nil
 }
+
+func (s *service) Update(ctx context.Context, def model.LabelDefinition) (model.LabelDefinition, error) {
+	if err := def.ValidateForUpdate(); err != nil {
+		return model.LabelDefinition{}, errors.Wrap(err, "while validating Label Definition")
+	}
+
+	ld, err := s.repo.GetByKey(ctx, def.Tenant, def.Key)
+	if err != nil {
+		return model.LabelDefinition{}, errors.Wrap(err, "while receiving Label Definition")
+	}
+
+	if ld.ID == "" {
+		return model.LabelDefinition{}, fmt.Errorf("id of received label definition is empty")
+	}
+
+	id := ld.ID
+	ld = &def
```

Comment:
Do we really want to override all the properties, or should we only allow to update schema? For example I would not update the tenant property.

Reply:
changed to update only schema